cni/README.md

```diff
@@ -1,15 +1,17 @@
 ## `cni` ##
 
-There are a wide variety of different [CNI][cni] network configurations. This
+There are a wide variety of different [CNI](https://github.com/containernetworking/cni) network configurations. This
 directory just contains an example configuration that can be used as the
 basis for your own configuration.
 
 To use this configuration, place it in `/etc/cni/net.d` (or the directory
 specified by `cni_config_dir` in your `libpod.conf`).
 
-In addition, you need to install the [CNI plugins][cni] necessary into
-`/opt/cni/bin` (or the directory specified by `cni_plugin_dir`). The
-two plugins necessary for the example CNI configurations are `portmap` and
-`bridge`.
+For example a basic network configuration can be achieved with:
 
-[cni]: https://github.com/containernetworking/plugins
+```bash
+sudo mkdir -p /etc/cni/net.d
+curl -qsSL https://raw.githubusercontent.com/containers/libpod/master/cni/87-podman-bridge.conflist | sudo tee /etc/cni/net.d/99-loopback.conf
+```
+
+Dependent upon your CNI configuration, you may need to install [CNI plugins](https://github.com/containernetworking/plugins) necessary for your configuration into `/opt/cni/bin` (or the directory specified by `cni_plugin_dir`).  Please refer to the [CNI](https://github.com/containernetworking) project page in GitHub for more information.
```

Comment:
there is no depending, you need it

Reply:
So is installing the port and bridge plugins sufficient?  Could I get away with:
Dependent on your CNI configuration, you will need to install as a minimum the port and bridge [CNI plugins](https://github.com/containernetworking/plugins) into `/opt/cni/bin`.